ep/build-velox/src/get_velox.sh

```diff
@@ -2,8 +2,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/zhli1142015/velox
+VELOX_BRANCH=add_flag_for_sort_in_window
```

Comment:
the change is merged to oap-project/velox

Reply:
Thanks, updated.